integration-testing/rnode_testing/rnode.py

```diff
@@ -176,71 +186,71 @@ def execution():
             process.join()
             raise TimeoutError(cmd, self.timeout)
 
-    def shell_out(self, *cmd, stderr=True):
+    def shell_out(self, *cmd: Union[str, int], stderr=True) -> str:
         exit_code, output = self.exec_run(cmd, stderr=stderr)
         if exit_code != 0:
             raise NonZeroExitCodeError(command=cmd, exit_code=exit_code, output=output)
         return output
 
-    def call_rnode(self, *node_args, stderr=True):
+    def call_rnode(self, *node_args: Union[str, int], stderr: bool = True) -> str:
```

Comment:
Similarly to my comments above:
```suggestion
    def call_rnode(self, *node_args: Union[str], stderr: bool = True) -> str:
```

Reply:
same above